subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/pkg/algorithm/hived_algorithm.go

```diff
@@ -869,6 +869,11 @@ func generatePodScheduleResult(
 	vc api.VirtualClusterName,
 	pod *core.Pod) internal.PodScheduleResult {
 
+	var allSuggestedNodes []string
+	for node := range suggestedNodeSet.Items() {
+		allSuggestedNodes = append(allSuggestedNodes, node.(string))
+	}
```

Comment:
Add a common func for set to convert to slice, so you can just toJson(the slice)?